src/components/EpisodePreview/EpisodePreview.jsx

```diff
@@ -0,0 +1,140 @@
+import React, { useContext } from "react";
+import {
+	EpisodesContext as Episodes
+} from "COMPONENTS/Episodes";
+import UTILS from "SHARED/utils.js";
+
+export default function EpisodePreview(props){
+
+	// CONTEXT
+	// ----------------------------
+	const {
+		get: {
+			episodeCount,
+			activeIndex,
+			overlayActive
+		},
+		set: {
+			activeIndex: setActiveIndex,
+			overlayActive: setOverlayActive
+		}
+	} = useContext(Episodes);
+
+	// RENDER
+	// ----------------------------
+	const { 
+		episodeId,
+		index,
+		number: episodeNo,
+		title,
+		thumbnail: {
+			src: thumbSrc,
+			description: thumbDesc
+		},
+		trailer: {
+			src: trailerSrc
+		},
+		prevTrailer: {
+			src: prevTrailerSrc
+		},
+		nextTrailer: {
+			src: nextTrailerSrc
+		}
+	} = props;
+
+	// handle visibility of active episode and episode preview overlay
+	const isActiveEpisode 	= activeIndex === index;
+	const isOverlayActive   = isActiveEpisode && overlayActive;
+	const hidden 			= !isOverlayActive;
+
+	const isFirst = index === 0;
+	const isLast  = index === episodeCount - 1;
+
+	const prevEpisodeNo = episodeNo - 1;
+	const nextEpisodeNo = episodeNo + 1;
+
+	const previewId = UTILS.convertToSafeString(title, "-") + "-preview";
```

Comment:
can we use a template literal for this? string concatenation just looks hella gross.

Reply:
Done.